tests/suites/create.js

```diff
@@ -267,6 +268,17 @@ export function createSuite({
             shouldHaveMandatoryPointers({proof});
           }
         });
+        it('Initialize components to an array that is the result of ' +
+        'CBOR-decoding the bytes that follow the three-byte BBS disclosure ' +
+        'proof header. If the result is not an array of five or six elements ' +
+        '— a byte array, a map of integers to integers, two arrays of ' +
+        'integers, and one or two byte arrays; an error MUST be raised and ' +
```

Comment:
This is a punctuation error, here and in the spec and anywhere else it occurs. A set-aside (like this list of elements) that is opened with an em-dash should also be *closed* with an em-dash, not closed with a semicolon.
```
         'integers, and one or two byte arrays — an error MUST be raised and ' +
```